examples/secrets-management-with-csi-secrets-driver-aws/secretconfig.yaml

```diff
@@ -0,0 +1,11 @@
+# This file represents `spec.parameters.objects` of [SecretProviderClass]
+# (https://docs.aws.amazon.com/secretsmanager/latest/userguide/integrating_csi_driver.html#integrating_csi_driver_SecretProviderClass)
+- objectName: "ARN OF THE SECRET OBJECT, which has username and password as objects as example"
```

Comment:
to test this we should create a secret and show that  its actually mounted and retrievable from within a pod. we can create the secrets manager secret within the terraform example and we'll need to use the [random provider](https://registry.terraform.io/providers/hashicorp/random/latest/docs) to generate a random name for the secret since secrets are unique per account/region and they stay around for a minimum of 7 days pending deletion

Reply:
Updated the examples to create secret on the fly. No need to use the random provider. With Terraform destroy, the secret also gets deleted.